nexus/tests/integration_tests/unauthorized.rs

```diff
@@ -322,6 +333,12 @@ static SETUP_REQUESTS: Lazy<Vec<SetupReq>> = Lazy::new(|| {
             body: serde_json::to_value(&*DEMO_CERTIFICATE_CREATE).unwrap(),
             id_routes: vec![],
         },
+        // Create a Support Bundle
+        SetupReq::Post {
+            url: &SUPPORT_BUNDLES_URL,
+            body: serde_json::to_value(()).unwrap(),
+            id_routes: vec!["/experimental/v1/system/support-bundles/{id}"],
```

Comment:
Note: If I want to get coverage for the download/index paths, I need them here as `id_routes` too.